pyproject.toml

```diff
@@ -15,6 +15,8 @@ optax = "^0.1.3"
 matplotlib = "^3.6.2"
 tqdm = "^4.64.1"
 numpy = "^1.23.4"
+transformers = "^4.28.1"
```

Comment:
Question, open to discussion. Do we want to depend on transformers by default? Or should we rather have it as an optional dependency, to enable if the user wants to use Hugging Face models and datasets?

Reply:
Having it as an optional-dependency is probably a better option